chapter5/solutions/ch5solutions.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    	<!--Use this link tag if you're using external CSS: <link rel="stylesheet" type="text/css" href="FILE_PATH_NAME.css">-->
+      <style>
+      #title{
```

Comment:
@JJ27 I think you need to import Google Fonts to use Raleway

Reply:
Nevermind I can just add it myself lol

Reply:
@phrdang Yeah you're right, it worked for me since I use chrome but I think for other browsers you need to import.

Reply:
The font issue should be all good now